features/endpoints/grafana/api/src/main/java/org/opennms/netmgt/endpoints/grafana/api/Dashboard.java

```diff
@@ -80,4 +81,33 @@ public void setUrl(String url) {
     public void setPanels(List<Panel> panels) {
         this.panels = panels;
     }
+
+    public List<String> getTags() {
+        return tags;
+    }
+
+    public void setTags(List<String> tags) {
+        this.tags = tags;
+    }
+
+    public DashboardMetadata getMeta() {
+        return meta;
+    }
+
+    public void setMeta(DashboardMetadata meta) {
+        this.meta = meta;
+    }
+
+    @Override
+    public String toString() {
+        return "Dashboard{" +
```

Comment:
Is there a reason for not using the `MoreObjects.ToStringHelper` instead?

Reply:
Nope. What advantage do we gain by using it though?

Reply:
Looks like we use MoreObjects in some of other beans in this package already. I'll switch it over...

Reply:
Yeah, I thought a while ago we agreed upon using `MoreObjects.toStringHelper` in order to be consistent throughout the code base.